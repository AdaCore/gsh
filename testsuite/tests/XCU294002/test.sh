count=1
while test $count -lt 1000; do
   echo "count = $count"
   count="${count}0"
done
echo $?

false
while false; do
   true
done
echo $?

