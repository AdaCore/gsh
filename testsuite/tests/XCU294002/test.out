count = 1
count = 10
count = 100
0
0
